physionet-django/user/templates/user/edit_credentialing.html

```diff
@@ -17,6 +17,7 @@ <h1>PhysioNet Credentialing</h1>
   <p><i class="fa fa-clock"></i> Your credentialing application was submitted on {{ current_application.application_datetime }}. We will respond shortly.</p>
 {% else %}
   <p>Your account is not credentialed. You may <a href="{% url 'credential_application' %}">apply for access</a>.</p>
+  <p><strong>If you have already been credentialed</strong>, please add the approved email to your profile.</p>
```

Comment:
"If your account has already been credentialed on the old PhysioNet site, please add the email address from the old site to this account. Your credentialed status will be automatically transferred when your email is verified."

I think we have to use the word 'old' here. Make it 100% clear.